### PR TITLE
Draw units on minimap

### PR DESCRIPTION
**Pull request by ThislsAUsername**
This was easier than I expected.

### CODE REVIEW

File: src/UI/Art/SpriteArtist/MiniMapArtist.java [hunk 1]
```diff
@@ -35,13 +36,14 @@ public static BufferedImage getMapImage(MapInfo mapInfo, int maxWidth, int maxHe
    * @param teamColors A set of colors to be used for drawing each team. If insufficient colors
    *        are provided, black will be used for any remaining ones. If null is passed in,
    *        the default team-color ordering will be used.
+   * @param factions Ditto, but factions and the default
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/MiniMapArtist.java:39]: Default what? Colors I assume, but we should say it.

File: src/UI/Art/SpriteArtist/MiniMapArtist.java [hunk 2]
```diff
@@ -89,29 +91,28 @@ private static BufferedImage generateFullMapImage(MapInfo mapInfo, Color[] teamC
...
             {
-              Color coColor;
-
-              // Log a warning if SpriteLibrary doesn't have enough colors to support this map.
-              if( co >= teamColors.length )
-              {
-                System.out.println("WARNING! '" + mapInfo.mapName + "' has more start locations than there are team colors!");
-
-                // But soldier onwards anyway.
-                coColor = Color.BLACK;
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/MiniMapArtist.java:107]: Are we just no longer worried about running out of colors, or is this case being handled in some other way?

File: src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java [hunk 3]
```diff
@@ -197,10 +204,15 @@ public BufferedImage update(PlayerSetupInfo info)
    * Draws little white boxes around all the currently-highlighted player's properties.
    * @param distance - defines how far the boxes should be from the property edge (serves to highlight the new properties when switching between players)
    */
-  private static void drawPlayerPropertyHighlights(Graphics g, int baseX, int baseY, MapInfo mapInfo, double scale, int faction, int distance)
+  private static void drawPlayerStuffHighlights(Graphics g, int baseX, int baseY, MapInfo mapInfo, double scale, int faction, int distance)
```
Comment (ThatIsAUsername) [inline src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java:207]: Yes. Stuff.

**Comment by ThatIsAUsername** [comment]
Neat.

File: src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java (hunk 3 above)
Comment (ThislsAUsername) [reply src/UI/Art/SpriteArtist/PlayerSetupTeamArtist.java:207]: `drawPlayerPropertyAndUnitHighlights` was just a smidge too long for my liking.

File: src/UI/Art/SpriteArtist/MiniMapArtist.java (hunk 2 above)
Comment (ThislsAUsername) [reply src/UI/Art/SpriteArtist/MiniMapArtist.java:107]: That logic is pushed into `getNthSpriteSpec()`

**Comment by ThatIsAUsername** [comment]
No further objections

**Pull request merged**